.golangci.yml

```diff
@@ -20,21 +20,6 @@ linters:
 linters-settings:
   goimports:
     local-prefixes: github.com/NVIDIA/k8s-device-plugin
-
-issues:
-  exclude-rules:
-  # We use math/rand instead of crypto/rand for unique names in e2e tests.
-  - path: tests/e2e/
-    linters:
-    - gosec
-    text: "G404"
-  # We disable the memory aliasing checks in tests
-  - path: ".*_test.go"
-    linters:
-    - gosec
-    text: "G601: Implicit memory aliasing in for loop"
-  # We create world-readable files in tests.
-  - path: ".*_test.go"
-    linters:
-    - gosec
-    text: "G306: Expect WriteFile permissions to be 0600 or less"
\ No newline at end of file
+  gosec:
```

Comment:
We simplify the config even further when we use `linters-settings` instead of `exclude-rules` to disable certain lint rules